docs/src/main/sphinx/release/release-455.md

```diff
@@ -0,0 +1,60 @@
+# Release 455 (28 Aug 2024)
+
+## General
+
+* Add query starting time in `QueryStatistics` in all [](admin-event-listeners).
+  ({issue}`23113`)
+* Add JMX metrics for the bean
+  `trino.execution.executor.timesharing:name=TimeSharingTaskExecutor` replacing
+  metrics previously found in `trino.execution.executor:name=TaskExecutor`.
+  ({issue}`22914`)
+* Fix memory tracking issue for aggregations that could cause worker crashes
+  with out-of-memory errors. ({issue}`23098`)
+* Fix loading of custom `CatalogStore` implementations. Set `catalog.store` to
```

Comment:
@dain and @hashhar - these release note entries feel like we really need to document this stuff .. correct? Are we ready to have this in the public docs?

Reply:
If you were implementing a custom CatalogStore loading it was impossible since the guice bindings were incorrect.

Reply:
Yeah ... thats my concern .. I kinda think there is no impact since we have not documented this catalog store stuff at all .. its just in the SPI but I am not sure we even have an example implementation

Reply:
I am taking this and the next entry out .. lets aim to get this feature documented as beta soon